Dockerfile

```diff
@@ -5,41 +5,20 @@ MAINTAINER Steve Chan sychan@lbl.gov
 
 ARG BUILD_DATE
 ARG VCS_REF
-
-RUN apt-get -y update && \
-    apt-get -y install wget ca-certificates git make gcc libc-dev libsasl2-dev && \
-    update-ca-certificates
-
-# This build of shock seems to malfunction if we go above Go 1.5
-RUN cd /usr/local && \
-    wget https://storage.googleapis.com/golang/go1.5.4.linux-amd64.tar.gz && \
-    tar xzf go1.5.4.linux-amd64.tar.gz && \
-    ln -s /usr/local/go/bin/go /usr/local/bin/go
-
-ENV GOPATH /tmp/goroot
+ARG BRANCH=develop
 
 RUN mkdir -p /kb/deployment/bin /kb/deployment/conf && \
-    mkdir -p /mnt/shock/site /mnt/shock/data /mnt/shock/logs && \
-    mkdir -p $GOPATH/src/github.com/MG-RAST  && \
-    go get github.com/pborman/uuid  && \
-    go get github.com/MG-RAST/golib  ; \
-    go get github.com/MG-RAST/go-dockerclient  && \
-    cd $GOPATH/src/github.com/MG-RAST  && \
-    git clone --recursive https://github.com/kbase/Shock -b auth2 && \
-    cp Shock/Makefile $GOPATH && cd $GOPATH && make version && \
-    cd $GOPATH/src/github.com/MG-RAST/Shock/shock-server  && \
-    go build  && \
-    cp shock-server /kb/deployment/bin/ && \
-    echo Shock server binary copied to /kb/deployment/bin
+    mkdir -p /mnt/shock/site /mnt/shock/data /mnt/shock/logs
```

Comment:
At the version of Shock that we are using, there wasn't a Dockerfile yet. They introduced it sometime after this ancient snapshot we're using. Their Dockerfile is much cleaner because the current codebase works properly with recent Go versions, so they base it on the alpine Go images. We are so far behind that apparently we can't use newer versions of Go, from what I recall.
If we merge upstream, we can just redo the dockerfile at that point.